### DISCUSSION TITLE
How to switch camera in react native SDK?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#6113] [FAQs]
I am currently utilizing the [WebRTC React Native SDK](https://github.com/ant-media/WebRTC-React-Native-SDK) for Android. My specific query pertains to the capability of switching the mobile camera between the front and rear options within the application.

Could you kindly provide guidance or suggestions on how to achieve this functionality?

Thank you.

**Comment by zerostack0** [comment]
Hi @Mohit-3196 

You can achieve it with the following code part

```
let isFrontCam = true;

try {
	// Taken from above, we don't want to flip if we don't have another camera.
	if ( cameraCount < 2 ) { return; };

	const videoTrack = adaptor.localStream.current.getVideoTracks()[0];
	videoTrack._switchCamera();

	isFrontCam = !isFrontCam;
} catch( err ) {
	// Handle Error
};
```
